in_toto/util.py

```diff
@@ -186,8 +186,8 @@ def import_public_keys_from_files_as_dict(filepaths, key_types=None):
 
 def import_gpg_public_keys_from_keyring_as_dict(keyids, gpg_home=False):
   """Creates a dictionary of gpg public keys retrieving gpg public keys
-  identified by the list of passed `keyids` from the gpg keyring at `gpg_home`.
-  If `gpg_home` is False the default keyring is used. """
+  identified by the list of passed `keyids` from a gpg keyring in `gpg_home`.
+  If `gpg_home` is False the default gpg home directory is used. """
```

Comment:
Thanks for disambiguating this. It makes me wonder whether we want to use `--keyring` instead later down the line though. What do you think?

Reply:
I wouldn't replace it because there is merit to overriding the homedir and we do this in tests a lot. But it would definitely be nice to add support for the`--keyring` option. Maybe worth a feature request in securesystemslib?